src/components/UserContext.tsx

```diff
@@ -1,13 +1,15 @@
 import React, { FC, useContext, useState } from 'react';
-import { IBasicUserInfo } from '../interfaces/IBasicUserInfo.interface';
 import { IProfile } from '../interfaces/IProfile.interface';
 import supabase from '../utils/supabase';
 
-const UserContext = React.createContext<IBasicUserInfo | null>(null);
-const UserUpdateContext = React.createContext<((newUser: IBasicUserInfo) => void) | null>(null);
-let loggedUser = supabase.auth.user() ?? null;
+const UserContext = React.createContext<IProfile | null>(null);
+const UserUpdateContext = React.createContext<((newUser: IProfile) => void) | null>(null);
+let loggedInUser = supabase.auth.user() ?? null;
 
 export const getUserInfo = async (uId: string) => {
```

Comment:
I personally really don't the naming of the 'uId' variable. On linux systems, the naming is uppercase and I suggest such or, eventually, a lower-case one.
```suggestion
export const getUserInfo = async (uid: string) => {
```

Reply:
Changed uId to uid.